### ISSUE TITLE
ENOENT Error with 2.2.2

### ISSUE THREAD

**Issue by gisu** [haydenbleasel/favicons#87]
Hi,
last week i found this nice Plugin, installed with Version 2.1.5 everything is fine. Today i started a new Build on CI, any new Build Fails because ENOENT

```bash
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open 'src/.system/favicon.html'
    at Error (native)
```

The Issue comes with 2.2.1 and 2.2.2. Don't know why exactly. When i switch back to 2.1.5 the build is fine.

**Comment by haydenbleasel** [comment]
Hey @gisu, can I see your Gulpfile? Also, obviously the issue states that src/.system/favicon.html doesn't exist, so can you confirm that the file does exist? (or are you trying to write it?)

**Comment by gisu** [comment]
Before i let the Plugin write the `favicon.html` in a empty `dist` Folder. With 2.1.5 without Issues. With 2.2.2 i become the Issue, so ein switched to a other Folder (that exist without a Build). I got the Issue also here, i added also a empty favicon.html so the plugin can overwrite - the same. File not found.

```js
gulp.task('build:favicon', function () {
  gulp.src(kittn.src.system + 'favicon.png')
    .pipe($.favicons({
      appName: pkg.title,
      appDescription: pkg.description,
      developerName: pkg.author.name,
      developerURL: pkg.author.url,
      background: '#fff',
      path: kittn.dist.cssimgRoot + 'system/',
      url: '',
      display: 'standalone',
      orientation: 'portrait',
      version: pkg.version,
      logging: false,
      online: false,
      icons: {
        ....
      },
      html: kittn.src.system + 'favicon.html'
    }))
    .pipe(gulp.dest(kittn.dist.cssimg + 'system/'));
});```

**Comment by haydenbleasel** [comment]
Hey @gisu, okay I'll look into this. I think I'll also add `mkdirp` as a dependency so we never get write errors if a file/folder doesn't exist.

**Comment by haydenbleasel** [comment]
Hey @gisu, I added `mkdirp` as a dependency so we never get write errors if a file/folder doesn't exist. Can you update and see what happens now?

**Comment by dandaka** [comment]
Probably the same bug.

Log https://gist.github.com/dandaka/d4cb03591267c7ac9bcb
Gulpfile and all other sources https://github.com/dandaka/dandaka.com/blob/master/gulpfile.js

**Comment by dandaka** [comment]
I have switched to 2.1.5. It generates images without errors, but favicon.html is empty.

**Comment by haydenbleasel** [comment]
I guess it's erroring when it doesn't find a file rather than ignoring that and writing a new one. I'll fix this up.

**Comment by dandaka** [comment]
Which file? favicons.html?

**Comment by haydenbleasel** [comment]
Whatever the output file is (so yep, `favicons.html`).

**Comment by dandaka** [comment]
Creating an empty file fixed a problem for me.

**Comment by haydenbleasel** [comment]
Fixed in `favicons@4.4.4`

**Comment by dandaka** [comment]
I get same error with `gulp-favicons@2.2.3`

```
events.js:142
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open './public/favicons/favicons.html'
    at Error (native)

[frontend-boilerplate] npm list gulp-favicons                         master  ✭
frontent-boilerplate@2.0.0 /Users/dandaka/projects/frontend-boilerplate
└── gulp-favicons@2.2.3
```

Creation of an empty file still fixes the problem.
